index.html

```diff
@@ -130,9 +130,17 @@ <h3 class="main-features__subtitle">Application Information</h3>
 	<footer class="footer">
 		<div class="footer-grid">
 			<div class="footer__item">
-				<a href="https://www.sap.com/" target="_blank">
-					<img src="./src/assets/images/SAP-logo.svg" alt="SAP Company Logo" width="96" height="48">
-				</a>
+				<ul class="footer-subsection__list footer-subselectionSAP">
```

Comment:
We are trying to follow the [BEM](https://getbem.com/) methodology. Please correct the `footer-subselectionSAP` class to something like `footer-subsection__list--copyright`.